src/pages/questionPage.js

```diff
@@ -61,16 +59,38 @@ function chooseAnswer() {
   answerElements.forEach((element) => {
     element.removeEventListener("click", chooseAnswer);
   });
+
+  document
+  .getElementById(NEXT_QUESTION_BUTTON_ID)
+  .addEventListener('click', nextQuestion);
 }
 
 // increment
 const incrementScore = (point) => {
   quizData.score += point;
+  // for local storage
+  if (quizData.score) {
+    const allScore = quizData.score;
+    localStorage.setItem("score", allScore); // setItem(key, value)
+  }
   document.getElementById(SCORE_DIV_ID).innerText = quizData.score;
 };
 
 const nextQuestion = () => {
-  quizData.currentQuestionIndex = quizData.currentQuestionIndex + 1;
+  if (quizData.currentQuestionIndex === quizData.questions.length - 1) {
+    document.getElementById(NEXT_QUESTION_BUTTON_ID).style.display = "none";
+    document.getElementById(SHOW_RESULT_ID).style.display = "block";
```

Comment:
Instead of doing this here, let's do it in the `questionView`. We need to conditionally display buttons!

Reply:
Alright

Reply:
We are not using it anymore, please ignore it we going to delete it on the last branch.